### ISSUE TITLE
Improved contrast between hovered and unhovered overlay icons

### ISSUE THREAD

**Issue by aubergine10** [CitiesSkylinesMods/TMPE#739]
In many tools, there is not much distinction between hovered and unhovered states of overlay icons.

Take this image from vehicle restrictions tool for example:

![img](https://user-images.githubusercontent.com/1386719/75257340-b89dea80-57dc-11ea-871d-29f126db62ed.png)

The hovered truck icon is not very different from the unhovered icons around it.

Suggestions:

* Hovered icons should be 100% opaque
    * I think this should be applied to all tools
* We could perhaps limit minimum overlay transparency (setting in mod options > general tab)
    * For example, if user sets 0% transparency all icons would be 100% opaque
    * So maybe have minimum transparency of 30%?
* Change size of icons when hovered (slightly bigger)?
    * This would certainly make them feel more responsive and aid context awareness

Also, regarding the overlay transparency mod option, it feels weird "dragging transparency up" (ie. the further right, the more transparent). Maybe the option should be renamed "Overlay opacity" and "dragging opacity down" (to the left = more transparent) would "feel" more natural from user perspective. (Note: It's potentially a western cultural thing, not sure).

**Comment by aubergine10** [comment]
Kian mentioend this in #721:

> When a tool is not selected but overlay for that tool is turned on in the options, icons of that tool are always rendered but a bit smaller than usual. If I make non-hovered Icons smaller how do you distinguish them with other icons from other tools?
